### ISSUE TITLE
Docs Site rendering fails when using expectations expect_column_values_to_not_be_in_set

### ISSUE THREAD

**Issue by cbuffevant** [fivetran/great_expectations#6963]
**Describe the bug**
The Site rendering is failing when using the expectation `expect_column_values_to_not_be_in_set`.

Not that useful log from Airflow 👇 (for a possible cause check Additional Notes).

```
[2023-01-27, 11:28:20 -03] {content_block.py:120} ERROR - An unexpected Exception occurred during data docs rendering.  Because of this error, certain parts of data docs will not be rendered properly and/or may not appear altogether.  Please use the trace, included in this message, to diagnose and repair the underlying issue.  Detailed information follows:
        ValidationError: "1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)".  Traceback: "Traceback (most recent call last):
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/content_block.py", line 109, in _render_list
    result = content_block_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/renderer.py", line 14, in inner_func
    return renderer_fn(*args, **kwargs)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/expectations/expectation.py", line 146, in inner_func
    ] = render_func(*args, **kwargs)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/expectations/core/expect_column_values_to_not_be_in_set.py", line 193, in _prescriptive_renderer
    renderer_configuration = RendererConfiguration(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer_configuration.py", line 172, in __init__
    super().__init__(**values)
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)
".
[2023-01-27, 11:28:21 -03] {content_block.py:120} ERROR - An unexpected Exception occurred during data docs rendering.  Because of this error, certain parts of data docs will not be rendered properly and/or may not appear altogether.  Please use the trace, included in this message, to diagnose and repair the underlying issue.  Detailed information follows:
        ValidationError: "1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)".  Traceback: "Traceback (most recent call last):
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/content_block.py", line 102, in _render_list
    result = content_block_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/validation_results_table_content_block.py", line 117, in row_generator_fn
    expectation_string_cell = expectation_string_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/renderer.py", line 14, in inner_func
    return renderer_fn(*args, **kwargs)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/expectations/expectation.py", line 146, in inner_func
    ] = render_func(*args, **kwargs)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/expectations/core/expect_column_values_to_not_be_in_set.py", line 193, in _prescriptive_renderer
    renderer_configuration = RendererConfiguration(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer_configuration.py", line 172, in __init__
    super().__init__(**values)
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)
".
[2023-01-27, 11:28:21 -03] {warnings.py:109} WARNING - /Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/validation_results_table_content_block.py:226: DeprecationWarning: V2 API style custom rendering is deprecated as of v0.13.28 and is not fully supported anymore; As it will be removed in v0.16, please transition to V3 API and associated rendering style
  warnings.warn(

[2023-01-27, 11:28:21 -03] {site_builder.py:508} ERROR - An unexpected Exception occurred during data docs rendering.  Because of this error, certain parts of data docs will not be rendered properly and/or may not appear altogether.  Please use the trace, included in this message, to diagnose and repair the underlying issue.  Detailed information follows:
                ValidationError: "1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)".  Traceback: "Traceback (most recent call last):
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/content_block.py", line 102, in _render_list
    result = content_block_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/validation_results_table_content_block.py", line 117, in row_generator_fn
    expectation_string_cell = expectation_string_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/renderer.py", line 14, in inner_func
    return renderer_fn(*args, **kwargs)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/expectations/expectation.py", line 146, in inner_func
    ] = render_func(*args, **kwargs)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/expectations/core/expect_column_values_to_not_be_in_set.py", line 193, in _prescriptive_renderer
    renderer_configuration = RendererConfiguration(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer_configuration.py", line 172, in __init__
    super().__init__(**values)
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/site_builder.py", line 472, in build
    rendered_content = self.renderer_class.render(resource)
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/page_renderer.py", line 125, in render
    sections = self._collect_rendered_document_content_sections(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/page_renderer.py", line 266, in _collect_rendered_document_content_sections
    sections += [
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/page_renderer.py", line 267, in <listcomp>
    self._column_section_renderer.render(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/column_section_renderer.py", line 450, in render
    remaining_evrs, content_block = self._render_table(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/column_section_renderer.py", line 438, in _render_table
    new_block = self._table_renderer.render(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/content_block.py", line 66, in render
    result = render_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/content_block.py", line 127, in _render_list
    result = content_block_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/validation_results_table_content_block.py", line 117, in row_generator_fn
    expectation_string_cell = expectation_string_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/validation_results_table_content_block.py", line 239, in expectation_string_fn_with_legacy_translation
    return legacy_expectation_string_fn(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer/content_block/expectation_string.py", line 23, in _missing_content_block_fn
    renderer_configuration: RendererConfiguration = RendererConfiguration(
  File "/Users/cesar/repos/airflow/lib/python3.9/site-packages/great_expectations/render/renderer_configuration.py", line 172, in __init__
    super().__init__(**values)
  File "pydantic/main.py", line 342, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for RendererConfiguration
__root__
  RendererConfiguration must be passed either configuration or result. (type=type_error.rendererconfiguration)
".
```

**To Reproduce**
1. I created an expectation suite using only that rule, and left everything else pretty much as default.
2. Run the suite.
3. Check the logs.

I'm more than happy to help to replicate the issue.

**Expected behavior**
Docs Sites should render when using this expectations.

**Environment (please complete the following information):**
 - Operating System: I replicated it in MacOS.
 - Great Expectations Version: develop branch.

**Additional context**

⚠️  I think the cause of the error is a typo in https://github.com/great-expectations/great_expectations/blob/develop/great_expectations/expectations/core/expect_column_values_to_not_be_in_set.py#L213

![image](https://user-images.githubusercontent.com/8312839/215123048-0b2aa8d9-0384-49ed-bbcd-f2f68d98db86.png)

**Comment by jmcorreia** [comment]
Suffering from the same, so I tried to do a contribution here -> https://github.com/great-expectations/great_expectations/pull/6990

**Comment by rdodev** [comment]
Hey @jmcorreia and @cbuffevant thanks for the report. Yes, it seems there was an issue with this expectation that several users were encountering. @jmcorreia kindly contributed a fix, but that won't make it to release until later in the week. If you're in a rush, perhaps consider using `develop` branch now that the PR was merged.

**Comment by cbuffevant** [comment]
Thanks @jmcorreia! @rdodev I think i can wait for the next release...

**Comment by talagluck** [comment]
Resolved by #6990
